test/test_firmware.py

```diff
@@ -41,7 +43,7 @@ async def test_begin_firmware_update_known_format(url, client_session, multisens
     ) as disconnect_mock:
         node = multisensor_6
         await begin_firmware_update(
-            url, node, "test", bytes(10), client_session, "test"
+            url, node, "test", bytes(10), client_session, "test", 0
```

Comment:
Side note: It's easier to read if we use keyword arguments when there are many arguments.

Reply:
noted